.renovaterc.json

```diff
@@ -43,6 +43,15 @@
       ],
       "automerge": true,
       "groupName": "GitLab"
+    },
+    {
+      "matchPackageNames": [
+        "pre-commit/mirrors-mypy"
+      ],
+      "matchManagers": [
+        "pre-commit"
+      ],
+      "versioning": "pep440"
```

Comment:
@lmilbaum it seems like since v1.0.0, mypy including the mirror, follow semver so it might not be necessary:

https://github.com/pre-commit/mirrors-mypy/tags

It likely wasn't picked up before due to the weird 0.xxx versioning.

But this won't hurt for sure.